be/CMakeLists.txt

```diff
@@ -211,6 +211,9 @@ set_target_properties(mysql PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/m
 add_library(hdfs STATIC IMPORTED)
 set_target_properties(hdfs PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libhdfs.a)
 
+add_library(fiu STATIC IMPORTED)
+set_target_properties(fiu PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libfiu.a)
```

Comment:
libfiu have some probability to deadlock in memory fault inection when running with tcmalloc in high pressure.
As for the jemalloc, I think the problem is the same. You need to take care to use it under high pressure.